src/Oci8/Schema/Grammars/OracleGrammar.php

```diff
@@ -168,7 +168,7 @@ public function compileTableExists()
      */
     public function compileColumnExists($table)
     {
-        return "select column_name from user_tab_columns where table_name = upper('" . $table . "')";
+        return "select column_name from all_tab_cols where table_name = upper('" . $table . "')";
```

Comment:
I think this is compiled on `public function getColumnListing($table)` and somewhere in the processor. Have to do office works now.. Will try to dig this further too when I had the chance. Thanks!